config.xml

```diff
@@ -99,4 +99,5 @@
         <variable name="CAMERA_USAGE_DESCRIPTION" value="The app needs to access your camera to scan barcodes" />
     </plugin>
     <plugin name="cordova-plugin-inappbrowser" spec="~1.7.1" />
+    <plugin name="cordova-plugin-wkwebview-engine" spec="https://github.com/ionic-team/cordova-plugin-wkwebview-engine.git"/>
```

Comment:
Does this add any size to the app binary? I remember adding Crosswalk added like 25 MB to the bundle size for Android.

Reply:
I built the app for production and it seems to be 4.5 MB, the same size as when I build it without the plugin.

Good call on checking that though.